releases/2020-11/js.md

```diff
@@ -34,9 +38,65 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 ## Release highlights
 
-### _Package name_
+---
+
+=== COPY THIS AND ADD THE INFORMATION OF YOUR PACKAGE: ===
+
+Keep in mind that:
+
+- Including the package name in the headers makes the URL links work for multiple packages.
+- The format of this file will be cleaned up once all of your proposals are in.
+
+---
+
+### _Project name_ 
+
+#### @azure/_package-name_@_version_ [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/<service-folder>/<package-folder>/CHANGELOG.md)
+
+(leave blank)
+
+##### Breaking Changes on @azure/_package-name_@_version_
+
+- _Add one or more, or remove the "Breaking Changes on ..." entire section._
+
+##### New Features on @azure/_package-name_@_version_
+
+- _Add one or more, or remove the "New Features on ..." section._
+
+##### Major Fixes on @azure/_package-name_@_version_
+
+- _Add one or more, or remove the "Major Fixes on ..." section._
+
+---
+
```

Comment:
```suggestion
### Core AMQP

#### @azure/core-amqp@2.0.0 [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/core/core-amqp/CHANGELOG.md#200-2020-11-12)

This release marks the general availability of the `@azure/core-amqp` version 2 package.
This release also includes the changes from versions `1.1.7` and `2.0.0-beta.1`.

##### Breaking Changes on @azure/core-amqp@2.0.0
- `satusDescription` in `CbsResponse` which is the output for `CbsClient.negotiateClaim()` is renamed to `statusDescription` to fix the spelling error.
- The `CbsClient.negotiateClaim()` method now takes the token string directly instead of the `AccessToken` object.

We are cleaning the public API surface by

- removing exports that are either not used by either `@azure/event-hubs` and `@azure/service-bus` packages (which are the two main consumers of this package)
  - AsyncLockOptions
  - executePromisesSequentially
  - Func
  - getNewAsyncLock
  - isNode
  - randomNumberFromInterval
  - Timeout
  - `DataTransformer` and `DefaultDataTransformer`.
    `dataTransformer` has been removed from `ConnectionContextBase` and `ConnectionContextBaseParameters`.
    This allows us to consider other forms of implementing serializers in the future.
  - `ConditionStatusMapper` and `MessagingErrorCodes` as these are only used internally by this package.
- Previously, `ConnectionConfig.validate()` overridden entityPath if `undefined` with `String(undefined) = "undefined"`. This has been updated to retain `undefined` in the validation.
- moving the classes/methods/interfaces that are very specific to Event Hubs/Service Bus to their corresponding packages.
  - SharedKeyCredential
  - EventHubConnectionConfig
- avoid re-exporting things from `rhea-promise` and `@azure/core-auth`
  - Dictionary
  - isAmqpError
  - Message
  - TokenCredential
  - isTokenCredential
  - AccessToken
- removing all IotHub related artifacts. These existed to support the IotHub support we had in Event Hubs v2 which has since been removed in Event Hubs v5 for a better separation of concerns
  - IotHubConnectionConfig
  - IotHubConnectionStringModel
  - IotSharedKeyCredential
  - isIotHubConnectionString
- removing all Event Hubs, Storage and Service Bus interfaces meant to be used with the `parseConnectionString()` method
  - ServiceBusConnectionStringModel
  - StorageConnectionStringModel
  - EventHubsConnectionStringModel

##### New Features on @azure/core-amqp@2.0.0
- `AmqpAnnotatedMessage` interface that closely represents the AMQP annotated message from the [AMQP spec](https://docs.oasis-open.org/amqp/core/v1.0/os/amqp-core-messaging-v1.0-os.html#section-message-format) has been added. New `AmqpMessageHeaders` and `AmqpMessageProperties` interfaces(properties with camelCasing) have been added in the place of re-exports from "rhea" library(properties with snake_casing).

##### Major Fixes on @azure/core-amqp@2.0.0
- Internal improvement - Previously, each `RequestResponseLink.sendRequest` call adds an "onMessage" listener to the `ReceiverEvents.message` event and keeps discarding the responses that did not match the request-id and returns the response if matched. Adding many listeners would also result in a warning such as `MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 message listeners added to [Receiver]. Use emittr.setMaxListeners() to increase limit`.
  This has been improved to reuse a single listener for all the requests by maintaining a map of deferred promises that would be resolved(or rejected) upon receiving a message event.

```

Reply:
Resolving since we don't mention core in the highlights.

Reply:
Updated based on this! I'll resolve.